crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -118,6 +120,16 @@ impl BloomSettings {
         composite_mode: BloomCompositeMode::EnergyConserving,
     };
 
+    /// A variation of [`NATURAL`](Self::NATURAL) that uses the [`Additive`](BloomCompositeMode::Additive)
+    /// composite mode.
+    ///
+    /// This will make emmissive materials appear to glow a lot stronger compared to
+    /// [`NATURAL`](Self::NATURAL).
+    pub const NATURAL_ADDITIVE: Self = Self {
```

Comment:
This is the new constant as suggested [here](https://github.com/bevyengine/bevy/pull/11981#issuecomment-1958578570). Please feel free to bikeshed the name :)

Reply:
Yeah, name seems fine. I can't immediately think of something better.